molgenis-app/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.1"
+services:
+    molgenis:
+      image: registry.molgenis.org/molgenis/molgenis-app:${TAG:-latest}
+      ports:
+        - "8081:8080"
+      environment:
+        - molgenis.home=/home/molgenis
+        - opencpu.uri.host=opencpu
+        - elasticsearch.transport.addresses=elasticsearch:9300
+        - db_uri=jdbc:postgresql://postgres/molgenis
+        - db_user=molgenis
+        - db_password=molgenis
+        - admin.password=admin
+        - "CATALINA_OPTS=-Xmx1g"
```

Comment:
In our documentation we do: ```CATALINA_OPTS="-Xmx2g -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"```

Reply:
Bit torn about the max heap space: In the tomcat 8.5 sop it still is 1g, and most VMs have 1g. And for most use cases 1g should be enough. Except for chr6 it currently is not.

Reply:
Will update the GC settings.